src/plan_expand_hypertable.c

```diff
@@ -115,6 +116,93 @@ const_datum_get_int(Const *cnst)
 	pg_unreachable();
 }
 
+/*
+ * Constify expressions of the following form in WHERE clause:
+ *
+ * column OP timestamptz - interval
+ * column OP timestamptz + interval
+ *
+ * Iff interval has no month component.
+ *
+ * Since the operators for timestamptz OP interval are marked
+ * as stable they will not be constified during planning a
+ * subset of these operations are immutable and can be safely
+ * constified during planning. We adjust those expressions here
+ * and transform them for those cases where it's safe to do so
+ * during planning.
+ */
+static OpExpr *
+constify_timestamptz_op_interval(PlannerInfo *root, OpExpr *constraint)
+{
+	Assert(op->args->length == 2);
+
+	Expr *left = linitial(constraint->args);
+	Expr *right = llast(constraint->args);
+	OpExpr *op;
```

Comment:
Nit: would suggest `opexpr` since it actually is an expression, not an operator (i.e., not a function).